script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -205,17 +207,20 @@ I accepted her words of concern with an open heart.
 [sha:53e2e7e45a8aedf543381f1fcd88f6fbd6fcb71f]{
 -- Page 16, Offset 1185.
 -- この七年間は本当に穏やかだった。それと同じように、自分のいた時間が有間の家の人たちにとって苦痛でなかったのなら、これほど喜ばしい事はない。
-The past seven years have been truly peaceful. As long as I hadn't caused trouble for the Arimas, I couldn't be happier.
+The past seven years have been truly peaceful for me.
+ And in the same way,
+ if the Arimas didn't feel bad with me around,
```

Comment:
"feel bad with" seems a bit off to me, and not sure about 'in the same way' here either - maybe something like
> If the Arimas didn't mind having me around either, then I couldn't be happier.